src/index.html

```diff
@@ -6,12 +6,173 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            name="feedback"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            name="feedback"
+          >
+        </label>
+
+        <label class="form-field">
+        How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+        Full date of birth:
+          <input
+            name="feedback"
```

Comment:
The input field for 'Full date of birth' should have a unique name, e.g. 'birthdate'.